va/va_vpp.h

```diff
@@ -435,6 +445,34 @@ typedef struct _VABlendState {
 #define VA_SOURCE_RANGE_REDUCED         1
 #define VA_SOURCE_RANGE_FULL            2
 
+/** @name Transfer function flags */
+/**@{*/
+/** \brief Transfer function none. */
+#define VA_TRANSFER_FUNCTION_NONE       0
+/** \brief Transfer function BT601 in SDTV. */
+#define VA_TRANSFER_FUNCTION_REC601     1
+/** \brief Transfer function BT709 in HDTV. */
+#define VA_TRANSFER_FUNCTION_REC709     2
+/** \brief Transfer function SRGB for Display Monitor. */
+#define VA_TRANSFER_FUNCTION_SRGB       3
+/** \brief Transfer function ST2084. */
+#define VA_TRANSFER_FUNCTION_ST2084     4
+/** \brief Transfer function BT1886. */
+#define VA_TRANSFER_FUNCTION_BT1886     5
+ /**@}*/
```

Comment:
Thanks for the feedback. #127 defines some colour primaries(BT2020, BT709, and BT601 etc. H.265 Spec Table E.3 Colour primaries). But here, the new enum are defining the transfer characteristics, H2.65 Table E.4 Transfer Characteristics. According to usage, I simplified Table E.4 into the above enum.

Reply:
#127 is a superset of this defining all three parts (colour primaries, transfer characteristics, matrix coefficients).

Unless you have specific reasons not to then I think it's a good idea to use the standard enums here rather than inventing new ones - the user can take the values directly from all codecs using them (MPEG-2, H.264, H.265, AV1, etc.), which avoids a translation step and allows forward compatibility.

Reply:
@fhvwy I agreed with you partially. I read #127, I just want to know how to handle HDR10(Color Primaries: BT2020, Transfer characteristics ST2084).